### PR TITLE
feat(orchestrator): add stale dependency cache fallback

### LINKED ISSUE #1828: feat(availability): add stale dependency cache fallback
## Description
Implement an LLM-friendly, independently shippable backlog item for **add stale dependency cache fallback**. This issue should stay narrow enough for one agent/PR while producing durable behavior, tests, and documentation where appropriate.

## Affected locations
- Schedulers, queues, health checks, and capacity policy
- External dependency handling and degradation paths
- Dashboard/status reporting

## Impact
Without this work, future agents and operators will keep rediscovering the same gap, creating avoidable coordination cost, reliability risk, or ambiguous handoffs.

## Recommendation
Implement the scoped behavior with deterministic tests, concise docs, and structured output that PM/liveness tooling can consume.

## Goals
Deliver add stale dependency cache fallback as a discrete capability or hardening change.
Make the behavior deterministic enough for autonomous workers to verify.
Keep the implementation small enough for a one-issue worker and one PR.
Capture operator/user-facing behavior in docs or help text when applicable.

## Expected outcomes
add stale dependency cache fallback is available, documented, and test-backed.
Future PM/worker handoffs can point at this issue without needing extra interpretation.
Failure modes are explicit and produce actionable messages instead of silent drift.
The change does not broaden into adjacent backlog work or unrelated refactors.

## Suggested implementation slices
Inventory the current code/docs paths that own this behavior.
Add or update the narrow model/config/schema needed for the feature or guardrail.
Implement the smallest runtime/docs change that satisfies the acceptance criteria.
Add focused fixtures/tests for success, failure, and edge cases.
Update docs/help output and handoff notes so later agents can use it.

## Acceptance criteria
- [ ] There is a dedicated implementation or documentation change for add stale dependency cache fallback.
- [ ] At least one automated test, fixture, or deterministic verifier covers the intended behavior.
- [ ] At least one negative/edge case is covered so the failure mode is explicit.
- [ ] Docs/help text or inline operator guidance explains how to use or interpret the behavior.
- [ ] The PR description links this issue and summarizes verification evidence.
- [ ] No unrelated production behavior is changed outside the stated scope.

## LLM handoff notes
- Treat this as a one-issue/one-PR unit.
- Start by searching for existing implementations and tests before adding new surfaces.
- Prefer structured outputs and fixtures over prose-only behavior.
- If this overlaps an existing open issue, comment with evidence and stop for PM deduplication instead of creating an unscoped implementation.
- Do not perform broad refactors or change unrelated worker/PM behavior while implementing this ticket.

## Source
Created by the 2026-07-11 fourth strategic backlog expansion run requesting 100 issues. Topic: **Availability**.

### PR DESCRIPTION
**Pull request by djm204**
## Summary
- preserve a previous known-good task output sidecar as a stale dependency cache
- rehydrate checkpointed dependency outputs from the stale cache when the primary sidecar is missing or corrupted
- emit warning/audit telemetry and document resume behavior

## Test Plan
- npm --prefix packages/franken-orchestrator test -- tests/unit/file-checkpoint-store.test.ts tests/unit/phases/execution.test.ts
- npm --prefix packages/franken-orchestrator run lint
- npm run build --workspace @franken/types && npm run build --workspace @franken/planner && npm run build --workspace @franken/observer && npm run build --workspace @franken/brain && npm run build --workspace @franken/critique && npm run build --workspace @franken/governor
- npm --prefix packages/franken-orchestrator run typecheck
- npm --prefix packages/franken-orchestrator run build

Closes #1828

### CODE REVIEW

**Comment by djm204** [comment]
@codex review

File: packages/franken-orchestrator/src/checkpoint/file-checkpoint-store.ts [hunk 1]
```diff
@@ -215,10 +219,48 @@ export class FileCheckpointStore implements ICheckpointStore {
...
+    }
+  }
+
   private get taskOutputDir(): string {
     return `${this.checkpointPath}.outputs`;
   }
 
+  private copyTaskOutputToStale(outputPath: string): void {
+    try {
+      copyFileSync(outputPath, `${outputPath}.stale`);
```
[the remark this answers is not in the document]
Comment (djm204) [reply packages/franken-orchestrator/src/checkpoint/file-checkpoint-store.ts]: Fixed in 33eb2001. The stale promotion path now creates the checkpoint parent before locking, validates/deserializes the primary sidecar before copying it to .stale, and leaves any existing known-good stale cache intact when the primary is corrupt. Added regression coverage for both the corrupt-primary preservation path and the nested-checkpoint first-unserializable-output path.

File: packages/franken-orchestrator/src/checkpoint/file-checkpoint-store.ts [hunk 2]
```diff
@@ -215,10 +219,48 @@ export class FileCheckpointStore implements ICheckpointStore {
...
+    } catch (error) {
+      if ((error as NodeJS.ErrnoException).code !== 'ENOENT') {
+        throw error;
+      }
+    }
+  }
+
+  private promoteTaskOutputToStale(outputPath: string): void {
+    this.withLock(() => {
```
[the remark this answers is not in the document]
Comment (djm204) [reply packages/franken-orchestrator/src/checkpoint/file-checkpoint-store.ts:267]: Fixed in 33eb2001. The stale promotion path now creates the checkpoint parent before locking, validates/deserializes the primary sidecar before copying it to .stale, and leaves any existing known-good stale cache intact when the primary is corrupt. Added regression coverage for both the corrupt-primary preservation path and the nested-checkpoint first-unserializable-output path.

**Comment by djm204** [comment]
@codex review
